### PR TITLE
fix: avoid onresize height loop [SE-4876]

### PR DESCRIPTION
**Pull request by arbrandes**
### Description 

Open edX sets a global `min-height: 100%` for the body element (a common strategy to allow for background images to fill the viewport), but this is undesirable for iframes where the `window.onresize` is used to programmatically adjust the iframe height.  Because `onresize` will fire when the body adjusts itself as per its `min-height: 100%`, in certain situations this will lead to an infinite resizing loop.  This is notably visible when:

* Rendering the video block preview when using Firefox: this causes the iframe   size to jiggle indefinitely.
* Rendering the HTML block preview when there are empty block elements (such as   `<p></p>`) that have a vertical margin set (which happens to be the default for HTML xblocks): this will cause the iframe to increase in  height infinitely.

By adding an inline `min-height: auto` on the wrapping body element, this overrides the Studio-served CSS, and thus fixes the problem.

### Test instructions

1. Deploy frontend-app-library-authoring in the devstack and create a library [as per the README](https://github.com/edx/frontend-app-library-authoring/#devstack-installation).
2. Create a new HTML block and edit its OLX by clicking on the "Source" tab, then include this content:
   ```
   <html>
     <p></p>
   </html>
   ```
3. Click save, then on the "View" tab.  You should **not** see the height of the iframe increase infinitely.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @arbrandes! I've created [OSPR-6083](https://openedx.atlassian.net/browse/OSPR-6083) to keep track of it in JIRA.

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Comment by natabene** [comment]
@arbrandes Thanks so much for your contribution. Is this ready for a review?

**Comment by arbrandes** [comment]
@natabene, yup, thanks!

**Review by jvdm — APPROVED** [review]
:+1:

- [x] I tested this: Verified correct behavior in the sandbox where the patch was deployed.
- [x] I read through the code
- n/a I checked for accessibility issues
- n/a Includes documentation
- n/a I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.

**Comment by arbrandes** [comment]
@bradenmacdonald, same here.  See SE-4876.

File: src/library-authoring/edit-block/LibraryBlock/wrap.js
Comment (bradenmacdonald) [inline src/library-authoring/edit-block/LibraryBlock/wrap.js:378]: Thanks for the nice comment directly in the code! ❤️

**Review by bradenmacdonald — APPROVED** [review]
:+1:

- [ ] I didn't tested this, already tested by earlier review
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] Includes documentation

**Comment by arbrandes** [comment]
Thanks, @bradenmacdonald!  Looks like I need an approval from somebody with explicit write access, though.  @kdmccormick, care to do the honors?  Pretty much a formality, since we already have two other approvals.

If you can also merge the twin change to to edx-platform in https://github.com/edx/edx-platform/pull/28875, that would be great.  (It really is the same change - it's just that unfortunately we need the xblock bootstrap code to live in two different places, currently.)

**Comment by kdmccormick** [comment]
Nice fix!

@arbrandes That "require approver with write access" rule is stricter than what we have enabled even for edx-platform, so I removed it. You should be good to merge this now!

As for edx-platform: alas, since I moved to tCRIL I don't have any visibility into edX.org production. I would go through your edX champion if you think that your change has risk of production impact.

**Comment by arbrandes** [comment]
Merging this, as it is both approved and extremely low-impact.

**Comment by edx-webhook** [comment]
@arbrandes 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-library-authoring&entry.867055334=https://github.com/openedx/frontend-app-library-authoring/pull/26&entry.1484655318=https://github.com/arbrandes&entry.752974735=2021-09-29+14:07&entry.1917517419=2022-02-04+15:04&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
